doc/howtos/backend/exercise-gantt

```diff
@@ -45,7 +45,7 @@ diff --git a/openacademy/views/openacademy.xml b/openacademy/views/openacademy.x
 +                <gantt string="Session Gantt" color="course_id"
 +                       date_start="start_date" date_delay="hours"
 +                       default_group_by='instructor_id'>
-+                    <field name="name"/>
++                    <!-- <field name="name"/> this is not required after Odoo 10.0 -->
```

Comment:
just remove it then?

Reply:
@mart-e https://github.com/odoo/odoo/pull/20342#discussion_r145961866

Reply:
👎